ui/app/AppLayouts/Profile/popups/RenameAccountModal.qml

```diff
@@ -21,10 +21,14 @@ StatusModal {
     id: popup
 
     property WalletStore walletStore
-    property var account
     property var emojiPopup
 
-    headerSettings.title: qsTr("Rename %1").arg(popup.account.name)
+    required property string accountName
+    required property string accountEmoji
+    required property string accountColorId
+    required property string accountAddress
```

Comment:
This property could be removed along with walletStore prop. Instead signal `accountUpdateRequested` signal could be added. The interaction with the store can be done in upper layer then.